selector-component.js

```diff
@@ -17,9 +17,11 @@ function register(module) {
       selected: '<brSelected',
       onSelect: '&brOnSelect',
       onAddItem: '&?brOnAddItem',
+      onDeselect: '&?brOnDeselect',
```

Comment:
I guess because it's available, we can call it and pass null whenever someone hits the deselect button, but that might be a little confusing because the user also has to handle onSelect cases when they transclude in the <br-select-options